src/inc/corcompile.h

```diff
@@ -727,6 +727,9 @@ enum CORCOMPILE_FIXUP_BLOB_KIND
     ENCODE_CHECK_FIELD_OFFSET,
 
     ENCODE_DELEGATE_CTOR,
+
+    ENCODE_GENERIC_DICTIONARY_ACCESS,
```

Comment:
The new values in this enum should be prefixes used to describe the kind of the lookup, something like `ENCODE_DICTIONARY_LOOKUP_TYPE` and `ENCODE_DICTIONARY_LOOKUP_METHOD`. They should be followed by regular fixup signature. For example, the signature for `typeof(List<T>)` should be:

`ENCODE_GENERIC_CONTEXT` `ENCODE_TYPE_HANDLE <typespec bytes for List<T> >`